### ISSUE TITLE
usage examples?

### ISSUE THREAD

**Issue by pracplayopen** [leprechaun/gocd-prometheus-exporter#6]
is there an example grafana dashboard that shows how you recommend using this?   

or a one-liner alert manager function that would tell you which pipeline is failing?

asking as we're sort of struggling with building useful graphs and alerts.   

for example to build an alert on any pipeline fail... would we use _latest_result?   if we did this, wouldn't it always get overwritten by any subsequent stage operation in any pipline?   

we could build alerts for every pipeline stage... but this seems like overkill both from creating the alerts and even more responding to them.   yes there can only be one stage fail per pipeline in general, but if multiple changes are pushed simulatenously (esp upstream) you can temporarily have the same pipeline failing in two different places (each using two different upstreams, or one different upstream and one material change in the pipeline).

perhaps this is user error on our part, so examples would be good at minimum.

what we were kind of hoping for is this would save us from writing our own exporter.   alerts and grafans and minimum need to correspond to business-level things... which in gocd isn't stages but pipelines. 

for example... if there is a easy way to get the status for an entire pipeline from a metric reference (rather than writing a promq operation that grouped all the specific dependencies together)... we'd love to know how to do this KISS-wise without having to be tied to the specific pipeline configuration at any one moment.   

right now we haven't even attempted this because it seems like if we make a gocd config change it could break graphs or alerts even though the pipeline concept was exactly the same.   the advantage of a pipeline though is it abstracts away from details of whats in it, at a team or business level at least.

so in summary, examples please and hopefully the general feedback on usage is... useful. :)
